tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-rocm.yml

```diff
@@ -100,16 +100,17 @@ stages:
                 /onnxruntime_src/tools/ci_build/build.py \
                   --config Release \
                   --use_rocm \
-                    --rocm_version=$(RocmVersion) \
-                    --rocm_home=/opt/rocm \
-                    --nccl_home=/opt/rocm \
+                  --rocm_version=$(RocmVersion) \
+                  --rocm_home=/opt/rocm \
+                  --nccl_home=/opt/rocm \
                   --update \
                   --parallel \
                   --build_dir /build \
                   --build \
                   --build_wheel \
                   --skip_tests \
-                  --enable_training
+                  --enable_training \
+                  --enable_training_torch_interop
```

Comment:
not sure whwat base image ROCM image is based on. If it is the centos manylinux, it might has no dev lib of Python installed. So for that case we disable UTs to makre sure build can pass. You can check this PR as a reference. https://github.com/microsoft/onnxruntime/commit/47b3ecb53b2fcde74cf6b7345b3407f6bc815f70

Reply:
Thanks a lot, it is based on centos, will check out the ref you mentioned.

Reply:
The simplest way to check is, you can firstly check whether all CIs pass or not. If not, and there is Python header or module not found issue, then it need disable the UTs.

Reply:
Hi, @pengwa @wschin - talked to Baiju, those model tests could be done in separate pipeline (i.e. AMD/anubis). However, since this is new option for the packaging, is there tests in CUDA pipelines already covered with the build with the option?

Reply:
in PR CIs, those tests will run for CUDA. We only disable them for packaging pipeline because those images are manylinux.

Reply:
I've enabled AMD CI to run ortmodule tests in this PR. This ROCm image is also manylinux so these tests are also disabled for the same reason.